components/network/openvpn/files/openvpn.xml

```diff
@@ -12,30 +14,51 @@
             <service_fmri value="svc:/system/filesystem/local"/>
         </dependency>
 
-        <method_context>
+        <method_context project=':default' resource_pool=':default' working_directory='/etc/openvpn'>
```

Comment:
The workdir should not influence an explicitly specified "atomic" config file, and should help with files referenced from it by relative locations - if they are all in /etc/openvpn per defaults. Although with the other bit (--cd) - the command-line option wins.